include/wtmpdb.h

```diff
@@ -41,6 +41,8 @@
 #define NSEC_PER_USEC ((uint64_t) 1000ULL)
 #define USEC_PER_SEC  ((uint64_t) 1000000ULL)
 
+#define LOGROTATE_DAYS 20
```

Comment:
I would use 60 days (~ 2 month) by default, then we can run it once a month and have the data of the last two month available by default.

Reply:
Yes, it is better.....